pallets/domains/src/tests.rs

```diff
@@ -0,0 +1,473 @@
+use frame_support::{assert_noop, assert_ok};
+use sp_runtime::{DispatchError, DispatchError::BadOrigin, traits::Zero};
+
+use pallet_parachain_utils::mock_functions::{another_valid_content_ipfs, invalid_content_ipfs, valid_content_ipfs};
+use pallet_parachain_utils::new_who_and_when;
+
+use crate::{DomainByInnerValue, Event, mock::*};
+use crate::Error;
+use crate::types::*;
+
+// `register_domain` tests
+
+#[test]
+fn register_domain_should_work() {
+    const LOCAL_DOMAIN_DEPOSIT: Balance = 10;
+
+    ExtBuilder::default()
+        .domain_deposit(LOCAL_DOMAIN_DEPOSIT)
+        .build()
+        .execute_with(|| {
+            let owner = account_with_balance(DOMAIN_OWNER, BalanceOf::<Test>::max_value());
+            let expected_domain = default_domain();
+            let expected_domain_lc = default_domain_lc();
+
+            assert!(get_reserved_balance(&owner).is_zero());
+
+            assert_ok!(_register_default_domain());
+
+            assert_eq!(Domains::domains_by_owner(&owner), vec![expected_domain_lc.clone()]);
+
+            let domain_meta = Domains::registered_domain(&expected_domain_lc).unwrap();
+            assert_eq!(domain_meta, DomainMeta {
+                created: new_who_and_when::<Test>(DOMAIN_OWNER),
+                updated: None,
+                expires_at: ExtBuilder::default().reservation_period_limit + 1,
+                owner: DOMAIN_OWNER,
+                screen_name: expected_domain.clone(),
+                content: valid_content_ipfs(),
+                inner_value: None,
+                outer_value: None,
+                domain_deposit: LOCAL_DOMAIN_DEPOSIT,
+                outer_value_deposit: Zero::zero()
+            });
+
+            assert_eq!(get_reserved_balance(&owner), LOCAL_DOMAIN_DEPOSIT);
+
+            System::assert_last_event(Event::<Test>::DomainRegistered(
+                owner,
+                expected_domain,
+            ).into());
+        });
+}
+
+#[test]
+fn register_domain_should_fail_with_bad_origin() {
```

Comment:
change name for force_register_....

Reply:
Yeah, still have to refactor tests a lot, because of these PR changes.